src/test/java/com/commercetools/sync/products/utils/ProductSyncUtilsTest.java

```diff
@@ -148,16 +145,79 @@ public void buildActions_FromDraftsWithMultipleDifferentValues_ShouldBuildUpdate
             AddExternalImage.ofVariantId(1,
                 Image.of("https://53346cfbf3c7e017ed3d-6de74c3efa80f1c837c6a988b57abe66.ssl.cf3.rackcdn.com/"
                     + "new-image-2.png", ImageDimensions.of(0, 0), null), true),
-            SetPrices.of(1,
-                asList(PriceDraft.of(BigDecimal.valueOf(1.18), DefaultCurrencyUnits.EUR)
-                                 .withChannel(Channel.referenceOfId("channel-key_1")),
-                    PriceDraft.of(BigDecimal.valueOf(1), MoneyImpl.createCurrencyByCode("EGP"))
-                              .withChannel(Channel.referenceOfId("channel-key_1")))),
+            RemovePrice.of(Price.of(MoneyImpl.ofCents(108, EUR))
```

Comment:
wow! this test looks like an all-in-one 😄. Too many asserts and setup code. I'm not sure how easy will be to maintain this test in the future

Reply:
It is indeed because it's testing the all-in-one util `ProductSyncUtils#buildActions` which does all of these things.